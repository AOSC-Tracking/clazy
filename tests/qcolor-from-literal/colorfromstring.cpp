#include <QtGui/QColor>

extern void takingColor(QColor);

void test()
{
    takingColor(QColor::fromString("#123")); // Warning
}
