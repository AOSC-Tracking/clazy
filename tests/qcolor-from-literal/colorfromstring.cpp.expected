qcolor-from-literal/colorfromstring.cpp:7:17: warning: The QColor ctor taking RGB int value is cheaper than one taking string literals [-Wclazy-qcolor-from-literal]
